### PR TITLE
[CCTRI-1585] Create a UI template of the Module Maker

### PR DESCRIPTION
**Pull request by AnastasiiaRozlyvan**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (mr-pavlenko) [inline public/index.html:9]: Do you really need this meta tag?

File: src/container/Content/Content.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import {connect} from "react-redux";
+import styles from "./Content.module.scss";
+import MarkdownEditor from "./components/MarkdownEditor";
+import ExternalReferences from "./components/ExternalReferences/ExternalReferences";
+import Properties from "./components/Properties/Properties";
+import ConfigurationSpec from "./components/ConfigurationSpec/ConfigurationSpec";
+import Capabilities from "./components/Capabilities/Capabilities";
+import * as Constants from "../../globals/constants/constants";
+import {updateSingleInput, updateFlags, onFileLoaded} from "../../redux/actions";
```
Comment (mr-pavlenko) [inline src/container/Content/Content.js:10]: space

File: src/container/Content/Content.module.scss [hunk 3]
```diff
@@ -0,0 +1,81 @@
+@import "src/globals/constants/colors";
+
```
Comment (mr-pavlenko) [inline src/container/Content/Content.module.scss:2]: you should use nesting.

File: src/container/Content/components/MarkdownEditor.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import ReactMde from "react-mde";
+import * as Showdown from "showdown";
+import "react-mde/lib/styles/css/react-mde-all.css";
+import { updateSingleInput } from "../../../redux/actions";
+import {connect} from "react-redux";
```
Comment (mr-pavlenko) [inline src/container/Content/components/MarkdownEditor.js:6]: space

File: src/container/Content/components/Properties/SupportedAPI.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+class SupportedAPI extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            title: this.props.api.title,
+            class: this.props.api.class
+        }
+    }
+
+    changeHandler = (event) => {
```
Comment (mr-pavlenko) [inline src/container/Content/components/Properties/SupportedAPI.js:16]: handlers should start with onSomeAction

File: src/container/Content/components/Sidebar/Sidebar.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+import {connect} from "react-redux";
+
+
+class Sidebar extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    save = () => {
+        let data = this.props.syncJSON;
```
Comment (mr-pavlenko) [inline src/container/Content/components/Sidebar/Sidebar.js]: Why did you use let?

File: public/index.html (hunk 1 above)
Comment (AnastasiiaRozlyvan) [reply public/index.html:9]: No. Removed.

File: src/container/Content/Content.js (hunk 2 above)
Comment (AnastasiiaRozlyvan) [reply src/container/Content/Content.js]: Done

File: src/container/Content/components/MarkdownEditor.js (hunk 4 above)
Comment (AnastasiiaRozlyvan) [reply src/container/Content/components/MarkdownEditor.js]: done

File: src/container/Content/components/Sidebar/Sidebar.js (hunk 6 above)
Comment (AnastasiiaRozlyvan) [reply src/container/Content/components/Sidebar/Sidebar.js]: Yes it's a mistake, replaced let with const.

File: src/container/Content/components/Properties/SupportedAPI.js (hunk 5 above)
Comment (AnastasiiaRozlyvan) [reply src/container/Content/components/Properties/SupportedAPI.js]: Fixed

File: src/container/Content/Content.module.scss (hunk 3 above)
Comment (AnastasiiaRozlyvan) [reply src/container/Content/Content.module.scss]: Done

**Pull request merged**